packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.module.css

```diff
@@ -5,3 +5,10 @@
   justify-content: center;
   text-transform: uppercase;
 }
+
+.modal iframe {
```

Comment:
does their modal not accept passed in styles?

Reply:
As far as I know, it does not. It _does_ expose a `heightChanged` callback, but it fires about a dozen times during loading, so I don't know that it'll be super useful unless we implement some kind of debouncing.